ONSdigital/eq-survey-runner
# Add blinker dependency and upgrade other dependencies

Fixes #1417

### What is the context of this PR?
Added blinker as a main dependency i.e. not just a dev dependency. Blinker was added in #1391 but not picked up because development Python modules are used for local development. 

### How to review 
- Create a new pipenv and run `pipenv install` (not `--dev`)
- Run the app and check it works
